doc/source/operations/upgrading-openstack.rst

```diff
@@ -54,6 +112,24 @@ using Heat, and disable the service.
 
 TODO: guide for disabling Heat
 
+Designate sink disabled by default
+----------------------------------
+
+Designate sink is optional designate service which listens for event
+Notifications, primarily from Nova and Neutron. It is disabled by default (when
+designate is enabled) in Caracal.
+
+This behavior can be overridden manually:
+
+.. code-block:: yaml
+   :caption: ``kolla/globals.yml``
+
+   designate_enable_notifications_sink: true
+
+If you are using Designate and do not make this change, the Antelope
+``designate-sink`` container will remain on the controllers after the upgrade.
+It must be removed manually.
```

Comment:
Looks like this was disabled by default simply because it's not required for Designate to work. I've proposed a wording change to make this clearer.

```suggestion
This was disabled by default simply because it isn't required for Designate to function. Unless you are using this feature, you do not need to make this change. Note that the Antelope
``designate-sink`` container will remain on the controllers after the upgrade.
It must be removed manually.
```

Reply:
I've gone with slightly different wording, what do you think?

Reply:
Alex's update is good to me